doc/requirements-and-limitations.md

```diff
@@ -30,8 +30,7 @@ The `SUPER` privilege is required for `STOP SLAVE`, `START SLAVE` operations. Th
 
 - MySQL 5.7 `JSON` columns are not supported. They are likely to be supported shortly.
 
-- The two _before_ & _after_ tables must share some `UNIQUE KEY`. Such key would be used by `gh-ost` to iterate the table.
-  - As an example, if your table has a single `UNIQUE KEY` and no `PRIMARY KEY`, and you wish to replace it with a `PRIMARY KEY`, you will need two migrations: one to add the `PRIMARY KEY` (this migration will use the existing `UNIQUE KEY`), another to drop the now redundant `UNIQUE KEY` (this migration will use the `PRIMARY KEY`).
+- The two _before_ & _after_ tables must share some `UNIQUE KEY`. Such key would be used by `gh-ost` to iterate the table. See [Read more](shared-key.md)
 
 - The chosen migration key must not include columns with `NULL` values.
```

Comment:
> I might even want to say "the chosen migration key must include only not null columns".

However that is not the case. By default `gh-ost` does indeed only look for Uinique Keys with `not null` columns, but supports `null`-able columns when the user takes responsibility and provides with `--allow-nullable-unique-key`.

For the migration to run correctly there must be no `null` values in the chosen key. `gh-ost` utilizes table definition when possible to _know_ that is not the case, or trusts the user when this is not the case.